tests/manage/z_cluster/test_flexible_scaling_ui.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_no_lso,
+    tier1,
+    vsphere_platform_required,
+    skipif_ui_not_support,
+    bugzilla,
+)
+from ocs_ci.ocs.resources.pod import get_osd_pods
+from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.add_replace_device_ui import AddReplaceDeviceUI
+from ocs_ci.ocs.node import get_node_names
+from ocs_ci.deployment.vmware import VSPHEREBASE
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.pv import get_pv_objs_in_sc
+from ocs_ci.ocs.cluster import check_ceph_health_after_add_capacity
+
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_no_lso
+@bugzilla("1943280")
+@vsphere_platform_required
+@skipif_ui_not_support("add_capacity")
+class TestFlexibleScalingUI(ManageTest):
+    """
+    Test Flexible Scaling via UI
+
+    """
+
+    def test_flexible_scaling_ui(self, setup_ui):
+        """
+        Test Procedure:
+        1.Add new disk to worker node
+        2.Add capacity via UI
+        3.Verify all OSD pods move to Running state
+        4.check ceph health after add capacity
+
+        """
+        log.info("Check the number of osd pods")
+        num_osd_pods = len(get_osd_pods())
+
+        log.info("Choose one worker node")
+        nodes = get_node_names()
+        nodes = [nodes[0]]
```

Comment:
You can use `random` here.

Reply:
I prefer to use the the first node because we can add a node without OCS label.